src/select-menu/src/OptionsList.js

```diff
@@ -153,16 +154,38 @@ export default class OptionsList extends PureComponent {
   }
 
   handleKeyDown = e => {
-    if (e.keyCode === 38) {
-      this.handleArrowUp()
-    }
-
-    if (e.keyCode === 40) {
-      this.handleArrowDown()
-    }
-
-    if (e.keyCode === 13) {
-      this.handleEnter()
+    if (e.key !== undefined) {
```

Comment:
We don't need to support this case – `event.key` has wide browser support all the way back to IE9.

Reply:
Okay. Will remove the code.

Reply:
removed keyCode piece